docquality.rst

```diff
@@ -69,15 +69,12 @@ Helping with the Developer's Guide
 
 .. highlight:: bash
 
-The Developer's Guide uses the same process as the main Python documentation,
-except for some small differences.  The source lives in a `separate
-repository`_.  Bug reports and patches should be submitted to the `Python
-bug tracker`_ using the ``devguide`` component.  Changes to the devguide
+The Developer's Guide has already migrated to GitHub, where the source lives in a `separate
```

Comment:
The status of the migration work is already covered in [PEP 512](https://www.python.org/dev/peps/pep-0512/) and it would be nice to not to repeat it here.